cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyMetrics.java

```diff
@@ -58,4 +63,14 @@ public long numSelfHealingStarted() {
   public long ongoingAnomalyDurationMs() {
     return _ongoingAnomalyDurationMs;
   }
+
+  @Override
+  public String toString() {
+    return String.format("{meanTimeBetweenAnomalies:{%s:%s, %s:%s, %s:%s}, "
+                         + "meanTimeToStartFix:%s, numSelfHealingStarted:%d, ongoingAnomalyDuration=%s}",
+                         GOAL_VIOLATION, toDurationString(_meanTimeBetweenAnomalies.get(GOAL_VIOLATION)),
+                         BROKER_FAILURE, toDurationString(_meanTimeBetweenAnomalies.get(BROKER_FAILURE)),
+                         METRIC_ANOMALY, toDurationString(_meanTimeBetweenAnomalies.get(METRIC_ANOMALY)),
```

Comment:
These may throw an exception if `_meanTimeBetweenAnomalies.get(...)` is `null`. Can we add a sanity check in constructor?